sklearn/metrics/ranking.py

```diff
@@ -250,7 +254,8 @@ def _binary_roc_auc_score(y_true, y_score, sample_weight=None):
             raise ValueError("Only one class present in y_true. ROC AUC score "
                              "is not defined in that case.")
 
-        fpr, tpr, tresholds = roc_curve(y_true, y_score,
+        # use closure for pos_label parameter
```

Comment:
Ah I see, Then you could do `partial(_binary_roc_auc_score, pos_label=pos_label)` and pass that as the `binary_metric`.